build/docker/github.dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ghcr.io/kmesh-net/kmesh-build:latest AS builder
```

Comment:
Why need a new dockerfile?Why not modify the original dockerfile?

Reply:
This is a temparary fix. Becase that @weli-l found before https://github.com/kmesh-net/kmesh/pull/742, it failed to build using docker 18.09